test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3654,12 +3654,14 @@ class GatherModule(torch.nn.Module):
             def __init__(self):
                 super(GatherModule, self).__init__()
                 self.register_buffer("weight", torch.ones(5))
+                self.embed = torch.nn.Embedding(8, 3)
```

Comment:
Please add some comments explaining:
1. How does this test relate to onnx::Gather and constant folding.
2. How does torch.nn.Embedding specifically relate to it. That is what condition is this meant to test for.